cmd/selectionMCMC/selectionMCMC.go

```diff
@@ -57,6 +58,8 @@ func main() {
 		UnPolarized:   *unPolarized,
 		Derived:       *derived,
 		Ancestral:     *ancestral,
+		FixedSigma:    *fixedSigma,
+		D:             1, //D is hardcoded as 1 for now. This represents the size of the ascertainment subset.
```

Comment:
I am not totally sure what this value captures.  For us is it the number of sites where hg38 does not match the HCA?

Reply:
This refers to ascertainment correction, where the condition for ascertainment was divergence or nondivergence between 1 human and 1 ancestor sequence, thus D=1.